### ISSUE TITLE
[FEATURE] Config for disabling condensing for tags

### ISSUE THREAD

**Issue by NVolcz** [Riverside-Healthcare/djLint#365]
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## Feature Request

I would like to have a configuration option for disabling the formatter's collapsing/condensing of certain tags into a single line.
Maybe an open like `"do_not_condense_tags": "for,if"` and a option for disabling it all together `"condense_lines": "false"`

The line I have and how I would like it to look like:
```
{% for e in elements %}
  {{ renderElement(e) }}
{% endfor %}
```
but the formatter turns it into this:
```{% for e in elements %}{{ renderElement(e) }}{% endfor %}```

**Comment by christopherpickering** [comment]
Thanks, this will be a great option to have. Probably an option for both of what you suggest - Disable it altogether, or just for specific tags. Same for html tags.

**Comment by NVolcz** [comment]
I found a bug today related to this.
The following code
```
{% set hero_content %}{% block hero %}{% endblock %}{% endset %}
```
gets formatted into:
```
{% set hero_content %}
  {% block hero %}{% endblock %}
{% endset %}
```
which gets messed up later when the `hero_content` variable is used:
```
{% if hero_content %}
  HERO STUFF HERE!
{% endif %}tk
```

**Comment by keikoro** [comment]
I'm not sure how the last comment relates to the original feature request, but wanted to ask if you're accepting PRs for the current codebase at all, @christopherpickering. I didn't take a super close look at the info available for v2.0.0 but it sounded like it may be a (partial) rewrite.

I'm assuming the idea here – for the option applicable to all tags – would be to allow disabling of condensation, as suggested, rather than changing the default and allowing _enabling_ of condensation.

**Comment by christopherpickering** [comment]
@keikoro yep, 2.0 is a big rewrite. I would be happy to accept pr's on the current branch, thanks!

**Comment by keikoro** [comment]
Thanks!

**Comment by monosans** [comment]
Fixed since v1.39.0
